toonz/sources/toonz/comboviewerpane.cpp

```diff
@@ -105,16 +105,6 @@ ComboViewerPanel::ComboViewerPanel(QWidget *parent, Qt::WFlags flags)
 	ImageUtils::FullScreenWidget *fsWidget = new ImageUtils::FullScreenWidget(this);
 	fsWidget->setWidget(m_sceneViewer = new SceneViewer(fsWidget));
 
-#ifdef Q_OS_WIN
-	//  Workaround for QTBUG-48288
-	//  This code should be removed after updating Qt.
-	//  Qt may crash in handling WM_SIZE of m_sceneViewer in splash.finish(&w)
-	//  in main.cpp. To suppress sending WM_SIZE, set window position here.
-	//  WM_SIZE will not be sent if window poistion is not changed.
-	::SetWindowPos(reinterpret_cast<HWND>(m_sceneViewer->winId()),
-		HWND_TOP, 0, 0, 0, 0, SWP_NOMOVE | SWP_NOSIZE);
-#endif
-
```

Comment:
You could consider replacing like:
```diff
- #ifdef Q_OS_WIN
+ #if QT_VERSION >= 0x050000 && QT_VERSION < 0x050600
```
and let it remain here for a while instead of promptly remove for all Win users.